metadata.yaml

```diff
@@ -17,7 +17,7 @@ doc_title: "Conda development guide."
 
 # Author names, ordered as a list. Each author name should be formatted as 'First Last'
 authors:
-  - "Telescope and Site Software"
+  - "Eric Coughlin"
```

Comment:
I think should be a list of people. So, I would recommend keeping your name as an author. Maybe add myself, Michael Reuter and Rob, as we all worked somehow to get to the state where we are now. Up to you....